src/interfaces.ts

```diff
@@ -500,7 +501,18 @@ export interface BaseInterest {
   name: string;
 }
 
```

Comment:
Switch relationships to base (i.e. Account[] -> BaseAccount[]). 
Remove Organization and organizationId